documents/src/pages/elements/tree.md

```diff
@@ -286,7 +286,7 @@ if (tree) {
 
 ## Data property interface
 
-The `data ` property that passes through the `ef-tree` uses the [TreeData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree/helpers/types.ts) interface for its data items.
+The `data ` property that passes through the `ef-tree` uses the [TreeData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree/helpers/types.ts) interface that extend from [Data Item](./custom-components/utils/data-management#data-item) interface.
```

Comment:
I thinkthis paragraph is describe wrong interface
ref:
```javascript
export type TreeData<T extends TreeDataItem = TreeDataItem> = T[] | CollectionComposer<T> | null;
```

Reply:
Fixed by the same idea as TreeSelect context.